lib/tasks/data.rake

```diff
@@ -6,4 +6,23 @@ namespace :data do
       UpdateSchoolsDataFromSourceJob.perform_later
     end
   end
+
+  desc 'Migrate phase to phases'
+  namespace :phase do
+    task migrate: :environment do
+      Rails.logger.debug("Running phase migration task in #{Rails.env}")
+
+      Subscription.all.each do |subscription|
+        search_criteria = subscription.search_criteria_to_h
+
+        next unless search_criteria.key?('phase')
+        next if search_criteria.key?('phases')
+
+        search_criteria['phases'] = [search_criteria['phase']]
+        search_criteria.delete('phase')
+
+        subscription.update!(search_criteria: search_criteria.to_json)
```

Comment:
We probably need to think about how we return these in the audit spreadsheet too

Reply:
Done a join on them.